server/middleware/validateUser.js

```diff
@@ -57,6 +57,35 @@ class ValidateUser {
 
     return next();
   }
+
+  /**
+   * validate user input on signIn
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {func} next
+   *
+   * @return {void}
+   */
+  static signInDataValidation(req, res, next) {
+    const error = {};
+    if (!req.body.username || req.body.username.trim() === '') {
```

Comment:
Destructure these parameters `username`, `password` and then use them accordingly @justega247